NitroxClient/GameLogic/DevConsoleEventEntry.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.Helper;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class DevConsoleEventEntry
+    {
+        private readonly IPacketSender packetSender;
+
+        public DevConsoleEventEntry(IPacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public void Spawn(GameObject gameObject)
+        {
+            List<InteractiveChildObjectIdentifier> childIdentifiers = VehicleChildObjectIdentifierHelper.ExtractGuidsOfInteractiveChildren(gameObject);
+            ConstructorBeginCrafting TPacket = new ConstructorBeginCrafting("", GuidHelper.GetGuid(gameObject), new NitroxModel.DataStructures.TechType(CraftData.GetTechType(gameObject).ToString()), 0, childIdentifiers, gameObject.transform.position, gameObject.transform.rotation, "", new Vector3[] { }, new Vector3[] { });
```

Comment:
Suggestion due to change in base code
```suggestion
            ConstructorBeginCrafting TPacket = new ConstructorBeginCrafting(new NitroxId(), NitroxIdentifier.GetId(gameObject), new NitroxModel.DataStructures.TechType(CraftData.GetTechType(gameObject).ToString()), 0, childIdentifiers, gameObject.transform.position, gameObject.transform.rotation, "", new Vector3[] { }, new Vector3[] { });
```